drive/amazon/oauth.go

```diff
@@ -86,9 +87,9 @@ func getFreshToken(conf *oauth2.Config) (*oauth2.Token, error) {
 	v.Set("redirect_uri", redirectURI)
 
 	// Ask the user to authorize shade, receive authorization code
-	fmt.Println("Please visit this URL:")
-	fmt.Printf("%s?%s\n", authURL, v.Encode())
-	fmt.Println("Authorize shade and paste the 'localhost' URL you are redirected to here:")
+	log.Println("Please visit this URL:")
```

Comment:
This is output specifically for the user, intended to be consumed at the command line.  I think it should remain fmt.Println, which I realize is in direct violation of my earlier guidance about log.Print in libraries.